frontend/src/features/map_planning/layers/plant/components/PlantingElement.tsx

```diff
@@ -23,6 +25,17 @@ const placeTooltip = (plant: PlantsSummaryDto | undefined) => {
   showTooltipWithContent(extendedPlant.displayName.common_name);
 };
 
+/**
+ * UI Component representing a single plant on the map.
+ *
+ * A single plant consists of a _Konva Group_ which itself is composed of:
+ * - A _Konva Circle_ filled with a selection-dependent colour
+ * - A _Konva Image_ depicturing the plant or showing a fallback image
+ *
+ * @param planting - A plant's details used for rendering it.
```

Comment:
Is it plural or singular? What is it?

```suggestion
 * @param planting - A plant's detail used for rendering the plant.
```

```suggestion
 * @param planting - Plant's details used for rendering the plant.
```